src/patternfly/components/DataToolbar/data-toolbar.scss

```diff
@@ -0,0 +1,356 @@
+
+    > * {
+      --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar__group--m-filter-group--space-items);
+    }
+  }
+
+  // Search filter
+  .pf-m-search-filter {
+    --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar--m-search-filter--spacer);
+
+    > * {
+      --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar--m-search-filter--space-items);
+    }
+  }
+
+  // Overflow menu
+  .pf-m-overflow-menu > * {
+    --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar--m-overflow-menu--space-items);
+  }
+
+  // Bulk select
+  .pf-m-bulk-select {
+    --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar--m-bulk-select--spacer);
+  }
+
+  // Label
+  .pf-m-label {
+    --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar--m-label--spacer);
+
+    font-weight: var(--pf-c-data-toolbar--m-label--FontWeight);
+  }
+
+  // Total items
+  .pf-m-pagination {
+    margin-left: auto;
+
+    .pf-c-pagination {
+      flex-wrap: nowrap;
+    }
+  }
+
+  .pf-m-clear {
+    justify-self: end;
+  }
+
+  // If pagination is sibling of align-right
+  .pf-m-align-right ~ * {
+    margin-left: 0;
+  }
+
+  .pf-c-chip-group {
+    align-items: baseline;
+  }
+}
+
+.pf-c-data-toolbar__content,
+.pf-c-data-toolbar__group {
+  display: flex;
+  align-items: center;
+}
+
+// Set all spacer values to '--pf-c-data-toolbar--spacer' in order to change this variable value contextually
+.pf-c-data-toolbar__content {
+  padding: var(--pf-c-data-toolbar__content--PaddingTop) var(--pf-c-data-toolbar__content--PaddingRight) var(--pf-c-data-toolbar__content--PaddingBottom) var(--pf-c-data-toolbar__content--PaddingLeft);
+
+  & + & {
+    padding-top: 0;
+  }
+}
+
+// Group
+.pf-c-data-toolbar__group {
+  --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar__group--spacer);
+}
+
+// Item
+.pf-c-data-toolbar__item {
+  // reset spacer
+  --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar__item--spacer);
+}
+
+.pf-c-data-toolbar__group,
+.pf-c-data-toolbar__item {
+  margin-right: var(--pf-c-data-toolbar--spacer);
+
+  &:last-child:not([class*="pf-m-spacer"]) {
+    --pf-c-data-toolbar--spacer: 0;
+  }
+}
+
+// Content
+.pf-c-data-toolbar__content,
+.pf-c-data-toolbar__expandable-content {
+  flex-wrap: wrap;
+}
+
+// Expandable content
+.pf-c-data-toolbar__expandable-content {
+  position: absolute;
+  top: 100%;
+  right: 0;
+  left: 0;
+  z-index: var(--pf-c-data-toolbar__expandable-content--ZIndex);
+  display: none;
+  padding: var(--pf-c-data-toolbar__expandable-content--PaddingTop) var(--pf-c-data-toolbar__expandable-content--PaddingRight) var(--pf-c-data-toolbar__expandable-content--PaddingBottom) var(--pf-c-data-toolbar__expandable-content--PaddingLeft);
+  visibility: hidden;
+  background-color: inherit;
+  box-shadow: var(--pf-c-data-toolbar__expandable-content--BoxShadow);
+
+  @media screen and (min-width: $pf-global--breakpoint--lg) {
+    position: static;
+    box-shadow: none;
+  }
+
+  &.pf-m-expanded {
+    display: grid;
+    grid-template-columns: 1fr;
```

Comment:
do you need to define this?

Reply:
nice catch. I had it there to place `clear all filters` button to [auto]column: 2, but I don't think we need it.